ViAn/Project/videoproject.cpp

```diff
@@ -161,5 +161,8 @@ void VideoProject::delete_artifacts(){
         Bookmark* temp = it->second;
         temp->remove_exported_image();
     }
-}
+    for(auto it2 = analyses.begin(); it2 != analyses.end(); it2++){
+        AnalysisMeta temp2 = it2->second;
```

Comment:
This variable can be "am" as well. "temp" isn't the best name and "temp2" is worse.